### ISSUE TITLE
Branch cleanup

### ISSUE THREAD

**Issue by tpluscode** [dotnetrdf/dotnetrdf#4]
Mercurial has this unfortunate feature, where it's easy to end up with many dangling branches. Most of the 100 branches here have not been touched for ages. Likely some even predate the Mercurial repository migration.

Most of them are equal to master. I'd start by removing them. Then we can have a look at any unmerged changes in whatever remains

**Comment by kal** [comment]
Yep, I'll start with the cleanup of the merged branches as you suggest!

**Comment by kal** [comment]
OK, all of the merged branches should be gone now. There are still a bunch of old branches (old meaning not updated in the last 2 years), but it should be a little easier to go through this one by one and decide whether to keep / merge them or not.

**Comment by tpluscode** [comment]
I just looked through the branches and most quite a few are safe to remove. I'd like to proceed with removing any that develop is up to date with or which have no outstanding changes.

I'll open PRs for the rest so that we can decide what to do next.
